public/pages/CreateDetector/containers/CreateDetector.tsx

```diff
@@ -65,15 +85,24 @@ export default class CreateDetector extends Component<CreateDetectorProps, Creat
   };
 
   onCreateClick = async () => {
-    if (this.state.creatingDetector) {
+    const { creatingDetector, detector, fieldMappings } = this.state;
+    if (creatingDetector) {
       return;
     }
 
     this.setState({ creatingDetector: true });
-    const createDetectorRes = await this.props.services.detectorsService.createDetector(
-      this.state.detector
+    const createMappingsRes = await this.props.services.fieldMappingService.createMappings(
+      detector.inputs[0].detector_input.indices[0],
+      detector.detector_type,
+      fieldMappings
     );
 
+    if (createMappingsRes.ok) {
+      console.log('Field mapping creation successful');
```

Comment:
Not blocking for this draft, but was this a development comment that should be removed at some point?

Reply:
will remove in follow up PR